.travis.yml

```diff
@@ -8,4 +8,4 @@ before_install:
 script:
   - docker ps | grep -q radicale
```

Comment:
I don't really know travis, but in terms of scripts you can do `docker ps --filter="name=radicale" -q`

Reply:
Can't make it to work locally.
Each line should exit with 0, but `docker ps --filter...` always exits with 0.
I did not find a shorter trick that the current one.

Reply:
BTW, I did not know about the filter param. Thanks! :cool:

Reply:
Ah damn, having problems with it myself (been trying out travis!) this should works in bash but not in travis:
`[[ -z "docker ps --filter=\"name=gogs\" -q ]]`

The current way is more than fine, easier to read actually too.